integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioManagedCoherence.java

```diff
@@ -443,7 +446,7 @@ private static void createDomainCrAndVerify(String adminSecretName, String domIm
         .spec(new DomainSpec()
             .domainUid(domainUid)
             .domainHomeSourceType("Image")
-            .image(domImage)
+            .image(domImage).imagePullPolicy((IMAGE_PULL_POLICY))
```

Comment:
put ".imagePullPolicy((IMAGE_PULL_POLICY))" in a separate line to be consistent with others.

Reply:
done